Apps/Sandcastle/gallery/development/Picking Sublayer Regions.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">  <!-- Use Chrome Frame in IE -->
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="Use Viewer to start building new applications or easily embed Cesium into existing applications.">
+    <meta name="cesium-sandcastle-labels" content="Beginner, Showcases">
```

Comment:
If this is a Showcase, it doesn't belong in the Development folder.  I'd suggest moving it up one directory level.  Also remove the "Beginner" tag.  Labels are editable in the "Info" pulldown in Sandcastle.